### PR TITLE
repo-sync command

### PR DESCRIPTION
**Pull request by hectorespert**
Add repo-sync command to synchronize repository in another folder o server (rsync command)

### CODE REVIEW

**Comment by arcan1s** [comment]
It seems that the script has some inaccuracies (like writing to the log without appending). I'll look at your pr later, hope at this weekend.

**Comment by hectorespert** [comment]
I fix the the log append in the repo-daemon

File: sources/repo-sync [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+func_ver() {
+  echo -e "                                   repo-sync                                    "
+  echo -e "A set of scripts to work with your repository                                   "
+  echo -e "Version : 1.5.3                                                  License : GPLv3"
+  echo -e "Author :                                             Evgeniy Alexeev aka arcanis"
+  echo -e "E-mail :                                          esalexeev (at) gmail (dot) com"
+  exit 0
+}
+
+func_upload_new_packages() {
```
Comment (arcan1s) [inline sources/repo-sync]: this function is not used

File: sources/repo-sync [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    *                  ) error_mes "flag"                                                 ;;
+  esac
+  shift
+done
+
+# default values
+ARCHS="i686 x86_64"
+DBNAME="custom"
+REPODIR="${HOME}/arch/repo"
+SYMLINK="no"
```
Comment (arcan1s) [inline sources/repo-sync]: this variable is not used as well

**Comment by arcan1s** [comment]
ok, thanks. I've marked two unused elements. Could you please remove them from the source code and I'll merge your pr.

**Comment by hectorespert** [comment]
Removed

**Pull request merged**